pom.xml

```diff
@@ -299,5 +299,10 @@
       <artifactId>trec-car-tools-java</artifactId>
       <version>13</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.solr</groupId>
+      <artifactId>solr-solrj</artifactId>
+      <version>${lucene.version}</version>
+    </dependency>
   </dependencies>
```

Comment:
Move up to group with the rest of the Lucene dependencies?

Reply:
:+1: